content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
+
+================
+Loans management
+================
+
+Odoo lets you manage your loans to have a comprehensive list of all loans undertaken by your company
+in order to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast).
+Set up your amortization schedule—or import it—and let Odoo automatically handle monthly interest
+and principal adjustments such that your financial reports are always accurate with minimal effort.
+
+Create a new loan
+-----------------
+
+You can create a new loan by going to :menuselection:`Accounting --> Accounting --> Loans`. When
+creating a new loan, you can create your amortization schedule in three ways:
+
+- By importing it from a supported file;
+- By calculating it from a couple of values (e.g. the :guilabel:`Amount Borrowed`, the
+  :guilabel:`Duration`, etc.);
+- By manually filling in the lines of the schedule.
```

Comment:
```suggestion
- importing it from a supported file
- calculating it from multiple input values (e.g. the :guilabel:`Amount Borrowed`, the
  :guilabel:`Duration`, etc.) using the :guilabel:`Compute` smart button.
- manually filling in the lines of the schedule
```

Reply:
Compute is actually a normal button, not smart, I've just removed the smart from your suggestion

Reply:
Wow yeah it is. Don't know what I was thinking haha.